lib/config/virtual_container_host.go

```diff
@@ -228,7 +233,7 @@ func (t *VirtualContainerHostConfigSpec) SetIsCreating(creating bool) {
 
 // IsCreating is checking if this configuration is for one creating VCH VM
 func (t *VirtualContainerHostConfigSpec) IsCreating() bool {
-	return t.ExecutorConfig.ID == CreatingVCH
+	return strings.HasPrefix(t.ExecutorConfig.ID, CreatingVCH)
```

Comment:
the string "CreatingVCH" is used in old version's vch, it'd be better not to change that value. In addition, I'd like to use something special for VCH, to make sure it's less possible to be confused by any other product's VM attributes.